app/controllers/Checkout.scala

```diff
@@ -57,30 +57,32 @@ object Checkout extends Controller with LazyLogging with CatalogProvider {
 
       val testOnlyPlans = if (tpBackend == TouchpointBackend.Test) List(catalog.weeklyZoneB.plansWithAssociations) else List.empty
 
-      val contentSubscriptionPlans = List(
+      val paperSubscriptionPlans = List(
```

Comment:
I think the original name was better as digipack is not paper. There is a ContentSubscription trait that is implement by both paper and digipack